src/bp/core/database/tables/server-wide/migrations.ts

```diff
@@ -9,7 +9,7 @@ export class MigrationsTable extends Table {
       table.increments('id')
       table.string('name')
       table.integer('batch')
-      table.timestamp('migrated_time')
```

Comment:
@dmytropanontko , can you add a migration for this?

Reply:
I can't make it. Because when you run migration with incorrect columns you have the following error:

![image](https://user-images.githubusercontent.com/37536815/51995144-d5bd9800-24ba-11e9-81d7-3dc77894e3cd.png)